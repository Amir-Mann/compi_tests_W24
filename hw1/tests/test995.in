"\FsI<(e\KAb|x~LC*xng-IAJBDkq$Q+vNZi"Q"C&{!=Nc9[5;DLR1GK?QHWSu3jj45]M] <i;nCa=YTi