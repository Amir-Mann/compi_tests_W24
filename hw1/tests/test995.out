1 ID D
Error 
